libs/qCC_io/ShpFilter.cpp

```diff
@@ -836,9 +823,10 @@ ccMesh *createMesh(
 }
 
 //! Builds the patches that where read from a MultiPatch record of a Shapefile
-//! patches are build as meshes, each part is its own mesh
-//! The only supported Patches/Part types are Triangle_Fan & Triangle_Strip
-CC_FILE_ERROR buildPatches(
+/** Patches are buit as meshes, each part is its own mesh.
+	The only supported Patches/Part types are Triangle_Fan & Triangle_Strip
+**/
```

Comment:
Why switch to C-style comments?

Mixing them like this makes it more difficult to read.

Reply:
I thought '//!' was for the brief description but apparently it's not... The issue is that it's like that everywhere in the code...